packages/@react-spectrum/list/test/ListView.test.js

```diff
@@ -848,81 +850,87 @@ describe('ListView', function () {
     });
 
     describe('onAction', function () {
-      installPointerEvent();
-      it('should trigger onAction when clicking items with the mouse', function () {
+      it('should trigger onAction when clicking items with the mouse', async function () {
         let onSelectionChange = jest.fn();
         let onAction = jest.fn();
         let tree = renderSelectionList({onSelectionChange, selectionMode: 'multiple', onAction});
 
         let rows = tree.getAllByRole('row');
-        userEvent.click(rows[1], {pointerType: 'mouse'});
+        await user.click(rows[1]);
         expect(onSelectionChange).not.toHaveBeenCalled();
         expect(onAction).toHaveBeenCalledTimes(1);
         expect(onAction).toHaveBeenLastCalledWith('bar');
 
         let checkbox = within(rows[1]).getByRole('checkbox');
-        userEvent.click(checkbox);
+        await user.click(checkbox);
         expect(onSelectionChange).toHaveBeenCalledTimes(1);
         checkSelection(onSelectionChange, ['bar']);
 
         onSelectionChange.mockReset();
-        userEvent.click(rows[2], {pointerType: 'mouse'});
+        await user.click(rows[2]);
         expect(onSelectionChange).toHaveBeenCalledTimes(1);
         checkSelection(onSelectionChange, ['bar', 'baz']);
       });
 
-      it('should trigger onAction when clicking items with touch', function () {
+      it('should trigger onAction when clicking items with touch', async function () {
         let onSelectionChange = jest.fn();
         let onAction = jest.fn();
         let tree = renderSelectionList({onSelectionChange, selectionMode: 'multiple', onAction});
 
         let rows = tree.getAllByRole('row');
-        userEvent.click(rows[1], {pointerType: 'touch'});
+        await user.pointer({target: rows[1], keys: '[TouchA]'});
         expect(onSelectionChange).not.toHaveBeenCalled();
         expect(onAction).toHaveBeenCalledTimes(1);
         expect(onAction).toHaveBeenLastCalledWith('bar');
 
         let checkbox = within(rows[1]).getByRole('checkbox');
-        userEvent.click(checkbox);
+        await user.click(checkbox);
         expect(onSelectionChange).toHaveBeenCalledTimes(1);
         checkSelection(onSelectionChange, ['bar']);
 
         onSelectionChange.mockReset();
-        userEvent.click(rows[2], {pointerType: 'touch'});
+        await user.pointer({target: rows[2], keys: '[TouchA]'});
         expect(onSelectionChange).toHaveBeenCalledTimes(1);
         checkSelection(onSelectionChange, ['bar', 'baz']);
       });
 
-      it('should support long press to enter selection mode on touch', function () {
-        let onSelectionChange = jest.fn();
-        let onAction = jest.fn();
-        let tree = renderSelectionList({onSelectionChange, selectionMode: 'multiple', onAction});
-        userEvent.click(document.body);
+      describe('still needs pointer events install', function () {
```

Comment:
blocks like this aren't quite compatible with new User Events
this may reflect on our code or may be a bug in User Event, until that's resolved though, I've broken down some of the single calls to their actual event makeup that we rely on

jsdom still doesn't support pointer events, https://github.com/jsdom/jsdom/issues/2527
we may be able to move the call to install back to where it originally came from, didn't have time to check before sabbatical. it's useful where it is because it tells us where we are actively using it and there is a bug in User Event that I've addressed here https://github.com/testing-library/user-event/pull/1121

Reply:
bit of a shame that we can't get rid of the installPointerEvent, but we can handle setting that up + performing the requisite events calls via our triggerLongPress util I guess

Reply:
yeah, it'd be good to just enable pointerEvents everywhere,  i don't think any browser is still using the mouse/touch alternate code paths that we are primarily testing anymore anyways. but I decided that was a separate PR as well.